app/assets/stylesheets/views/_report-child-abuse.scss

```diff
@@ -0,0 +1,8 @@
+.phone-numbers-report-child-abuse {
+  padding-top: 15px;
+  padding-bottom: 25px;
```

Comment:
Rather than magic numbers, could we use the standard `$gutter` values? Or a 5px variant of?